data-prepper-plugins/translate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/translate/TranslateProcessor.java

```diff
@@ -40,15 +42,17 @@ public class TranslateProcessor extends AbstractProcessor<Record<Event>, Record<
     private static final Logger LOG = LoggerFactory.getLogger(TranslateProcessor.class);
     private final ExpressionEvaluator expressionEvaluator;
     private final TranslateProcessorConfig translateProcessorConfig;
-    private final LinkedHashMap<Range<Float>, String> rangeMappings;
-    private final Map<String, String> individualMappings;
-    private final Map<Pattern, String> compiledPatterns;
+    private final LinkedHashMap<Range<Float>, Object> rangeMappings;
+    private final Map<String, Object> individualMappings;
+    private final Map<Pattern, Object> compiledPatterns;
+    private final TypeConverter converter;
```

Comment:
Another option to not take a dependency on all mutate-events processors would be for us to just move the convert processor out to its own module

Reply:
I will do it in the next PR

Reply:
I think this translate processor needs to be moved into mutate-event-processors